### ISSUE TITLE
DawDreamer Can't work for Addictive drums2

### ISSUE THREAD

**Issue by heozzi** [DBraun/DawDreamer#67]
Hello

I rendered the Addictive drums2 through DawDreamer.
Result -> Render works, but no sound
Is it solvable?
We think preset is not connecting with samples that Addictive drums2 originally give.

Thanks
KB

**Comment by DBraun** [comment]
0.6.2 has a new way to open the UI, load, and save states of plugins. Can you give it a try? See `tests/test_plugins.py`

**Comment by kyungyunlee** [comment]
Works when I use the open_editor, but after I save the state and load the state, it doesn't render again... 😢

**Comment by DBraun** [comment]
0.6.5 might fix this and hopefully you can still use any state files you've already created.

**Comment by kyungyunlee** [comment]
Hmm, just updated the library. But it still doesn't work when loading the existing state..

**Comment by DBraun** [comment]
@kyungyunlee Could you try building/testing locally? https://github.com/DBraun/DawDreamer/wiki/Developer's-Guide and `build_macos.sh` in the root directory will help. Maybe this line https://github.com/DBraun/DawDreamer/blob/e3762e1c5cd86d24923d16201d0cf20b18630324/Source/PluginProcessor.cpp#L332 could also be placed inside `loadPlugin`. Something might need to force the plugin's editor to execute.

**Comment by luzpaz** [comment]
Any progress here ?

**Comment by DBraun** [comment]
No, sorry. Lately my main focus has been on the Faust-related features, especially JAX.
